### PR TITLE
Create configuration documents for Helm Broker

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Add configuration docs for Helm Broker
- Fix typos in Kyma configuration docs and HB docs
- Rearrange the existing `Configuration` doc into a tutorial

**Related issue(s)**
#4028

### CODE REVIEW

File: docs/helm-broker/05-02-etcd-stateful.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **etcd.secure** | Specifies whether TLS can be used. | `false` |
```
Comment (polskikiel) [inline docs/helm-broker/05-02-etcd-stateful.md]:
```suggestion
| **etcd.secure** | Indicates if TLS is used. | `false` |
```

File: docs/helm-broker/05-02-etcd-stateful.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+| **etcd.resources.limits.cpu** | Defines limits for CPU resources. | `200m` |
+| **etcd.resources.limits.memory** | Defines limits for memory resources. | `256Mi` |
+| **etcd.resources.requests.cpu** | Defines requests for CPU resources. | `50m` |
+| **etcd.resources.requests.memory** | Defines requests for memory resources. | `64Mi` |
+| **replicaCount** | Defines the number of etcd Pods in a cluster. | `1` |
```
Comment (polskikiel) [inline docs/helm-broker/05-02-etcd-stateful.md]:
```suggestion
| **replicaCount** | Defines the size of the etcd cluster. | `1` |
```

File: docs/helm-broker/05-04-configure-hb.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ By default, the Helm Broker fetches bundles listed in the `index.yaml` file from
 
     Then, run:
     ```bash
-    kubectl apply my-helm-repos-urls
+    kubectl apply -f my-helm-repos-urls
```
Comment (polskikiel) [inline docs/helm-broker/05-04-configure-hb.md]: we need to work with that example cuz its not correct

File: docs/helm-broker/01-01-helm-broker.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ title: Overview
...
 
-The Helm Broker fetches bundles which contain a set of specific [files](#details-create-a-bundle). You must place your bundles in a repository of an appropriate [format](#details-create-a-bundles-repository). By default, the Helm Broker fetches bundles from the release of the [`bundles`](https://github.com/kyma-project/bundles/releases) repository. You can also [configure](#configuration-configuration) the Helm Broker to fetch bundle definitions from any remote HTTP server.
+The Helm Broker fetches bundles which contain a set of specific [files](#details-create-a-bundle). You must place your bundles in a repository of an appropriate [format](#details-create-a-bundles-repository). By default, the Helm Broker fetches bundles from the release of the [`bundles`](https://github.com/kyma-project/bundles/releases) repository. You can also [configure](#configuration-configuration) the Helm Broker to fetch bundle definitions from any remote HTTPS server.
```
Comment (polskikiel) [inline docs/helm-broker/01-01-helm-broker.md]: revert that change pls

File: docs/helm-broker/05-01-helm-broker.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.cfgReposUrlName** | Specifies the name of the ConfigMap which provides the bundles repository URLs. | `helm-repos-urls` |
```
Comment (polskikiel) [inline docs/helm-broker/05-01-helm-broker.md]:
```suggestion
| **global.cfgReposUrlName** | Specifies the name of the default ConfigMap which provides the bundles repositories URLs. | `helm-repos-urls` |
```

File: docs/helm-broker/05-01-helm-broker.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+| **global.isDevelopMode** | Defines that the repository URL must be an HTTPS server. If set to `true`, HTTP servers are also acceptable.  | `false` |
```
Comment (polskikiel) [inline docs/helm-broker/05-01-helm-broker.md]:
```suggestion
| **global.isDevelopMode** | Defines that each repository URL must be an HTTPS server. If set to `true`, HTTP servers are also acceptable.  | `false` |
```

File: docs/helm-broker/01-01-helm-broker.md (hunk 4 above)
Comment (klaudiagrz) [reply docs/helm-broker/01-01-helm-broker.md]: To discuss.

**Pull request merged**